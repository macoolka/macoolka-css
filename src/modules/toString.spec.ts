import {theme,parse as _parse,toString}from './index';
const parse = _parse(theme);
describe('parse Border', () => {
     it('toString',()=>{
        expect(toString(parse({
            mkHeight:'full',
            mkTypography: 'h2',
            mkFlexAlign: 'center',
            mkMedia:[{mkColor:'red',width:200},{mkColor:'body',width:300},{mkColor:'body',width:500},{fontSize:'40px',width:700}],
            selector:{
                ':hover':{
                    mkTypography: 'h1',
                }
            }
        }))).toMatchSnapshot();
    })
});


