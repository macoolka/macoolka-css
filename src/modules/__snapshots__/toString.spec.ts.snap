// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse Border toString 1`] = `
"min-height: 100vh;
max-height: 100vh;
letter-spacing: -.00833em;
line-height: 3.75rem;
font-size: 60px;
width: 200px;
justify-content: center;
font-weight: 300;
text-decoration: inherit;
text-transform: inherit;
@media screen and (max-width: 80em) {
  mk-color: body;
  width: 300;
}
@media screen and (max-width: 93em) {
  mk-color: body;
  width: 500;
}
@media screen and (max-width: 120em) {
  font-size: 40px;
  width: 700;
}
:hover {
  letter-spacing: -.01562em;
  line-height: 6rem;
  font-size: 96px;
  font-weight: 300;
  text-decoration: inherit;
  text-transform: inherit;
}"
`;
